src/zgamma_producer.cpp

```diff
@@ -49,81 +46,154 @@ void ZGammaVarProducer::WriteZGammaVars(nano_tree &nano, pico_tree &pico, vector
         pico.out_llphoton_deta().push_back(fabs(dilep.Eta() - photon.Eta()));
         pico.out_llphoton_iph().push_back(igamma);
         pico.out_llphoton_ill().push_back(ill);
+	if(sig_jet_nano_idx.size() > 1) {
+	  TLorentzVector j1, j2, dijet;
+	  j1.SetPtEtaPhiM(nano.Jet_pt()[sig_jet_nano_idx[0]], nano.Jet_eta()[sig_jet_nano_idx[0]], nano.Jet_phi()[sig_jet_nano_idx[0]], nano.Jet_mass()[sig_jet_nano_idx[0]]);
+	  j2.SetPtEtaPhiM(nano.Jet_pt()[sig_jet_nano_idx[1]], nano.Jet_eta()[sig_jet_nano_idx[1]], nano.Jet_phi()[sig_jet_nano_idx[1]], nano.Jet_mass()[sig_jet_nano_idx[1]]);
+	  dijet = j1 + j2;
+	  pico.out_llphoton_dijet_dphi().push_back(llg.DeltaPhi(dijet));
+	  pico.out_llphoton_dijet_balance().push_back((dilep+photon+j1+j2).Pt()/(dilep.Pt()+photon.Pt()+j1.Pt()+j2.Pt()));
+	  pico.out_photon_jet_mindr().push_back(min(photon.DeltaR(j1), photon.DeltaR(j2)));
+	  pico.out_photon_zeppenfeld().push_back(abs(photon.Eta() - (j1.Eta() + j2.Eta())/2));
+	  pico.out_jet_pt1().push_back(j1.Pt());
```

Comment:
Do we really want to have a variable called jet_pt1 and jet_p2?

Reply:
We could drop them. I just added them for convenience if we want to use them while doing a dijet MVA training.